Pod/Core/TypeConversion.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  TypeConversion.swift
+//  Pods
+//
+//  Created by Sam Dean on 4/4/17.
+//
+//
+
+import Foundation
+
+
+public protocol FromStringable {
```

Comment:
`FromStringable` 😂  -- how about `StringRepresentable` 🤔 ? I don't know if I prefer the `static func` that returns an optional or a failable initializer e.g. would `String`, `Int`, `Double` and `Bool` "automatically" conform to a protocol like:

```
protocol StringRepresentable {
    init?(_ string: String)
}

extension String: StringRepresentable {} //works
extension Double: StringRepresentable {} //works
extension Bool: StringRepresentable {} //works
extension Int: StringRepresentable {} //doesn't work, needs an implementation
```

The reason the `Int` one doesn't work is because `Int`'s implementation is `public init?(_ text: String, radix: Int = default)`

I _think_ this looks a bit nicer, what do you think?